CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing Guide
+
+Welcome to the Viam TypeScript SDK. Please read this guide prior to opening a pull request.
+
+## Development setup
+
+Developing on the TypeScript SDK requires macOS, Linux, or WSL / Bash for Windows. To get started, clone this repository:
+
+```shell
+git clone https://github.com/viamrobotics/viam-typescript-sdk.git
+```
+
+### Prerequisites
+
+You'll need to make sure the following tools are installed on your machine and accessible in your `PATH`.
+
+- Node.js
+  - `node`
+  - `npm`
+- Protobuf
+  - `protobuf`
+  - `protoc-gen-grpc-web`
```

Comment:
I don't think this is extensive. I had to run the setup instructions from `viamrobotics/app` before I could get my local dev environment working, so I don't know what the minimum required setup is here

Reply:
hm, curious what that was. the only other dependency I can think of is `buf`, but we install that one locally to `bin/buf`

Reply:
oh, looks like `buf` is a available through npm now - might be easier to use that